plugins/modules/zos_copy.py

```diff
@@ -2014,8 +2111,10 @@ def allocate_destination_data_set(
     # empty dataset was created for the user by an admin/operator, and they don't have permissions
     # to create new datasets.
     # These rules assume that source and destination types are compatible.
+    # To not generate and None return to the main code generate an empty dict of dest_params to return
```

Comment:
So here you're creating an empty dictionary to maintain consistency of what gets returned inside this function? (always returning a boolean and a dictionary)

Reply:
The function is called one time [here](https://github.com/ansible-collections/ibm_zos_core/blob/a8ddfeabdfd91d392a512c7ba10f5d4a69afa082/plugins/modules/zos_copy.py#L2552)

Reply:
I still don't quite understand what this comment means.